docs/docker-compose.yml

```diff
@@ -3,7 +3,6 @@ version: "3.9"
 services:
 
   sphinx:
-    privileged: true
```

Comment:
Why is privileged unnecessary in this case?

Reply:
Oh thanks. This is a weird PR because this line was never present in any rootski code before sphinx, but I merged all the code to trunk to test a GitHub trigger. So it looks like this line was always here. Same-ish with your other comment. Although, I should clean that up.

But if you _really_ want to know, docker containers MUST run as `root` in order to write to any volume mounts in the GitHub actions environment. Initially I had created a `sphinx-docs` user in the container to create the draw.io diagrams, that required me to add `privileged: true`. That allowed me to run the sphinx-build container locally on my machine. But I had to take that out and use the `root` user once I started getting it to work in CI.

Reply:
Yeah, clean ups can be future. Incremental approaches are always good.